unicorn-core/src/test/scala/org/virtuslab/unicorn/repositories/UsersRepositoryDSLTest.scala

```diff
@@ -0,0 +1,265 @@
+package org.virtuslab.unicorn.repositories
+
+import org.scalatest.FlatSpecLike
+import org.scalatest.Matchers
+import org.scalatest.OptionValues
+import org.virtuslab.unicorn.WithId
+import org.virtuslab.unicorn.BaseTest
+import org.virtuslab.unicorn._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+trait DSLUserTable {
+
+  val unicorn: Unicorn[Long] with HasJdbcProfile
+  val identifiers: Identifiers[Long]
+
+  import unicorn._
+  import unicorn.profile.api._
+
+  object User extends EntityDsl(identifiers) {
```

Comment:
Don't you have to override Id, Row and Table as well? Seems like travis compiled this PR, but how?